gradio/routes.py

```diff
@@ -415,8 +416,11 @@ def main(request: fastapi.Request, user: str = Depends(get_current_user)):
 
         @app.get("/info/", dependencies=[Depends(login_check)])
         @app.get("/info", dependencies=[Depends(login_check)])
-        def api_info():
-            # The api info is set in create_app
+        def api_info(all_endpoints: bool = False):
```

Comment:
Initially, I had implemented it so that the `/info` route just returned all of the endpoints, even those with `show_api=False` and we filtered in the frontend. However, this led to very, very large payloads for complex apps like the theme builder causing slow load times / crashing. We should figure out a way to reduce the payload for `/info`, but in the meantime, here we don't load all endpoints by default -- only if you use the API Recorder.